app/imports/components/mainLayout.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import Header from '/imports/components/header.jsx';
+import Footer from '/imports/components/footer.jsx';
+
+export const MainLayout = ({header, content, footer}) => (
```

Comment:
no no no
they are in brackets cuz it is an object.
the prop expected by the function is a {} with those properties inside, is some ES6